apps/meteor/ee/app/livechat-enterprise/server/lib/LivechatEnterprise.js

```diff
@@ -215,6 +214,82 @@ export const LivechatEnterprise = {
 		await AutoCloseOnHoldScheduler.unscheduleRoom(roomId);
 		await LivechatRooms.unsetOnHoldAndPredictedVisitorAbandonmentByRoomId(roomId);
 	},
+
+	async saveDepartment(_id, departmentData, departmentAgents) {
```

Comment:
Could you add some JSDocs to the method parameters? That would improve typing from TS modules and let this method prepared for migration.

Reply:
This is a good thing 🤔 

@MartinSchoeler , can you handle that?
_id is an string
departmentData is an object with types found in `defaultValidations` (is _almost_ an ILivechatDepartment)
departmentAgents is an object with `upsert` `remove` keys, both having an array of strings (_ids) inside


🤗